api/server/models/user.js

```diff
@@ -68,7 +75,7 @@ module.exports = function (User) {
                                     platform: platform,
                                     device_id: device_id,
                                     token: token,
-                                    email_id: email_id,
+                                    email: email_id,
                                     name: name,
                                     password: hash_password,
```

Comment:
DONE : user will be create using User.create of loopback which itself encode the password.